contrib/gitian/DOCKRUN.md

```diff
@@ -74,12 +74,18 @@ You can set other options for that script by setting the OPT variable when runni
 e.g.
 
 ```bash
-OPT="-j 8" ./dockrun.sh v0.17.2.3
+OPT="-j 8" ./dockrun.sh v0.17.3.0
 ```
 
 You can also examine the build and install logs by running a shell in the container, e.g.
 
 ```bash
+# Tail running logs
+docker exec -it gitrun /bin/bash
+tail -f builder/var/*.log
+
+# Inspect logs
 docker exec -it gitrun /bin/bash
 more builder/var/install-linux.log
+more builder/var/build.log
```

Comment:
`build.log` is the name that gitian writes to while the build is running. When the build is finished it gets moved to build-<os>.log, so that's what you would want to check on a successful run. Of course, if it fails during the build then it would still be build.log.